script.js

```diff
@@ -0,0 +1,59 @@
+const savedList = document.getElementById('savedList');
+const doneList = document.getElementById('doneList');
+let savedListTitle = document.getElementById('savedListTitle');
+savedListTitle.innerHTML = '해야할 일';
+doneListTitle.innerHTML = '완료한 일';
+
+form.addEventListener('submit', (e) => {
+  //할 일 개수 세기
+  const countSavedNum = () => {
```

Comment:
countSaverNum, countDoneNum 함수 표현식(함수 포인터 할당) 부분은 따로 빼서 바깥에 위치하는것이 좋을것 같습니다. 함수는 하나의 기능만 하도록 최대한 간결하게 작성하는 것이 좋다고 알고있습니다!

Reply:
맞습니다! 또한 `addEventListener` 안에 직접 함수를 작성해 주셨는데, 이 부분을 따로 분리하는 게 가독성 측면에서도 더 좋을 것 같습니다.

```suggestion
  const handleFormSubmit = () => { 함수 내용 }
  form.addEventListener('submit', handleFormSubmit);
```

함수가 길어서 다 옮기진 못했지만 이런 식으로요!

Reply:
함수 분리하기 명심하겠습니다!! 두 분 다 리뷰 감사드립니다 :)